tests/schemas/api_v3/posValidators.schema.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+import regex from './regex';
+
+const punishmentPeriod = {
+	start: Joi.number().integer().min(0).required(),
+	end: Joi.number().integer().min(1).required(),
+};
+
+const sharingCoefficient = {
+	tokenID: Joi.string().pattern(regex.TOKEN_ID).required(),
+	coefficient: Joi.string().required(),
+};
+
+const validatorSchema = {
+	name: Joi.string().pattern(regex.NAME).required(),
+	totalStakeReceived: Joi.string().pattern(regex.POSITIVE_DIGITS).required(),
```

Comment:
`totalStakeReceived` is renamed to `totalStake`
```suggestion
	totalStake: Joi.string().pattern(regex.POSITIVE_DIGITS).required(),
```

Reply:
Applied.